### PR TITLE
Upstream/toastr

### PR DESCRIPTION
**Pull request by ChrisBr**
Very simple implementation of the timeout. It only notifies you if the last notification happened more than one minute ago.

### CODE REVIEW

**Comment by ChrisBr** [comment]
@ancorgs @imobach please review!

File: src/app/components/feed/feeds.factory.js [hunk 1]
```diff
@@ -29,6 +29,7 @@
       this.isPublisher = attrs.isPublisher || false;
       this.isLocalScreen = attrs.isLocalScreen || false;
       this.isIgnored = attrs.ignored || false;
+      this.notifierTimeout = 0;
```
Comment (ancorgs) [inline src/app/components/feed/feeds.factory.js]: I wouldn't discard more notifications in the future, so I'd use a less generic name. Let's say something like `speakMutedTimeout`

**Comment by ancorgs** [comment]
I miss resetting the status of the timeout if the user unmutes and mutes again.

File: src/app/components/feed/feeds.factory.js (hunk 1 above)
Comment (ChrisBr) [inline src/app/components/feed/feeds.factory.js]: Okay!

**Comment by ChrisBr** [comment]
@ancorgs Good point!

**Comment by ChrisBr** [comment]
@ancorgs probably I won't have time to finish it before the weekend .. jfyi

**Comment by ancorgs** [comment]
Don't worry. I'm on vacation anyways

**Comment by ancorgs** [comment]
Superseded by #53

**Pull request closed without merging**